spaceship/spec/provisioning_profile_spec.rb

```diff
@@ -54,6 +54,30 @@
     end
   end
 
+  describe '#all via xcode api' do
+    around(:all) do |example|
+      switch = ENV['SPACESHIP_AVOID_XCODE_API']
+      example.run
```

Comment:
Could you explain this further? I'm not clear on what case a `rescue` will satisfy.